inventory.py

```diff
@@ -25,13 +25,20 @@
 SUBSCRIPTION_NAME = ENV['API_SUBSCRIPTION_NAME']
 API_SCOPE_PREFIX = ENV['API_SCOPE_PREFIX']
 MAX_REQ_ATTEMPTS = ENV['MAX_REQ_ATTEMPTS']
+PER_PAGE_COUNT = ENV['PER_PAGE_COUNT']
```

Comment:
Maybe we should also plug this into my request function?

Reply:
Perhaps